curl-sys/build.rs

```diff
@@ -212,6 +212,11 @@ fn main() {
             .file("curl/lib/vauth/vauth.c");
     }
 
+    if cfg!(not(windows)) {
```

Comment:
I think you'll need to check env vars instead of `cfg!` here because of cross-compilation.